utilities/persistent_cache/persistent_cache_test.cc

```diff
@@ -29,6 +29,12 @@ static void OnOpenForRead(void* arg) {
       std::bind(OnOpenForRead, std::placeholders::_1));
 }
 
+static void OnDeleteDir(void* arg) {
+  char* dir = static_cast<char*>(arg);
+  std::string cmd("rm -r -f " + std::string(dir));
```

Comment:
Ok. I got lazy around implementing a recursive delete and testing across platforms. Let me do it anyways.